batch-setup/make_tiles.py

```diff
@@ -102,7 +102,7 @@ def vpc_of_sg(sg_id):
              'missing-meta-tiles-write'):
     config = {
         'logging': {
-            'config': 'logging.conf.sample'
+            'config': 'batch-setup/logging.conf.sample'
```

Comment:
Should we have the full path to the logging config (i.e: `os.path.join(os.path.dirname(os.path.realpath(__file__)), 'logging.conf.sample')` or similar), so that it doesn't matter what the working directory is?

Reply:
Yup, I'm always running all commands from the tileops root anyway because I'm not sure if there would be other problems, but we certainly don't lose anything by using the full path and it's easy to do. db0cecb